packages/core/src/lib/GooglePlay.ts

```diff
@@ -58,4 +61,53 @@ export class GooglePlay {
     await this.gradleWrapper.executeGradleCommand(
         ['publishBundle', '--artifact-dir', filepath, '--track', track]);
   }
+
+  /**
+   * This calls the Google Play Console through the nodejs api and then calls bundles.list which
+   * lists all current Android App Bundles of the app and edit which should help us narrow down the
+   * highest uploaded app bundle in the play console. This works irrespective of whether this is a
+   * Chrome OS or Android only release.
+   */
+  async getLargestVersion(
+      packageName: string,
+      serviceAccountJsonFilePath: string,
+  ): Promise<number> {
+    if (!this._googlePlayApi) {
+      this._googlePlayApi = this.getAndroidClient(serviceAccountJsonFilePath);
+    }
+    const edit = await this._googlePlayApi.edits.insert({packageName: packageName});
```

Comment:
So I don't know anything about the Google Play APIs, but it seems a bit surprising that we need to use `edits` to just get a list of packages. If we somehow crash (or the user terminates execution) between line 81 and line 89, have we left some permanent change on the server?

Reply:
Yes, I found this a bit odd as well in my research. Fourtunately, edits automatically [come with a timeout](https://developers.google.com/android-publisher/api-ref/rest/v3/edits?hl=zh-TW#AppEdit), so if its not committed, it should just expire and not leave dangling edits on the server.